lib/bolt/analytics.rb

```diff
@@ -139,6 +141,15 @@ def report_bundled_content(mode, name)
         end
       end
 
+      def plan_counts(project_path)
```

Comment:
This function can kind of go anywhere. I opted to put it here since we're using it for analytics and putting it here indicates that, not because it relies on anything in this class.